src/soilbiogeochem/SoilBiogeochemCarbonFluxType.F90

```diff
@@ -199,6 +214,21 @@ subroutine InitHistory(this, bounds, carbon_type)
         call hist_addfld1d (fname='HR', units='gC/m^2/s', &
              avgflag='A', long_name='total heterotrophic respiration', &
              ptr_col=this%hr_col)
+!KO
```

Comment:
doesn't seem like these are being written out, or they are all 0. should they be removed from the code?